libvast/vast/defaults.hpp

```diff
@@ -352,10 +352,13 @@ constexpr size_t initially_requested_ids = 128;
 constexpr std::chrono::milliseconds telemetry_rate = std::chrono::milliseconds{
   1000};
 
-// Interval between checks whether a signal occured.
+/// Interval between checks whether a signal occured.
 constexpr std::chrono::milliseconds signal_monitoring_interval
   = std::chrono::milliseconds{750};
 
+/// Time after which a request is considered to have failed.
+constexpr std::chrono::seconds request_timeout = std::chrono::seconds{10};
```

Comment:
This sounds very generic, maybe consider `caf_request_timeout` to make clear which kinds of requests are meant here.

Reply:
I've been thinking about `node_request_timeout`. I don't like the `caf_` prefix, as it makes it sound like it's a CAF thing.